docs/hub/datasets-argilla.md

```diff
@@ -0,0 +1,87 @@
+# Argilla
+
+Argilla is a collaboration platform for AI engineers and domain experts that require high-quality outputs, full data ownership, and overall efficiency.
+
+## Importing and exporting datasets and records
+
+This guide provides an overview of how to import and export your dataset or its records to the Hugging Face Hub.
+
+In Argilla, you can import/export two main components of a dataset:
+- The dataset's complete configuration defined in `rg.Settings`. This is useful if your want to share your feedback task or restore it later in Argilla.
+- The records stored in the dataset, including `Metadata`, `Vectors`, `Suggestions`, and `Responses`. This is useful if you want to use your dataset's records outside of Argilla.
+
+🚀 Try the Argilla 🤝 Hugging Face [deployment integration on Spaces](https://huggingface.co/docs/hub/en/spaces-sdks-docker-argilla)!
+
+## Prerequisites
+
+First [login with your Hugging Face account](../huggingface_hub/quick-start#login):
+
+```bash
+huggingface-cli login
+```
+
+Make sure you have `argilla>=2.0.0` installed:
+
+```bash
+pip install -U argilla
+```
+
+## Push an Argilla dataset to the Hugging Face Hub
```

Comment:
maybe give some context that argilla's goal is to create datasets and so this is the step when the dataset is ready ?

feel free to add a screenshot as well :)